ocp_install_env.sh

```diff
@@ -6,6 +6,7 @@ export CLUSTER_DOMAIN="${CLUSTER_NAME}.${BASE_DOMAIN}"
 export SSH_PUB_KEY="${SSH_PUB_KEY:-$(cat $HOME/.ssh/id_rsa.pub)}"
 export NETWORK_TYPE=${NETWORK_TYPE:-"OpenShiftSDN"}
 export EXTERNAL_SUBNET=${EXTERNAL_SUBNET:-"192.168.111.0/24"}
+export MIRROR_IP=${MIRROR_IP:-"192.168.111.1"}
```

Comment:
Should this use the provisioning network instead, e.g. 172.22.0.1?

Reply:
Good point,  probably it should because although currently the IPA ramdisk will download from the hosted cache, when we remove that it'll need to download from this mirror over the provisioning network.